### DISCUSSION TITLE
First time LoRA training

### DISCUSSION THREAD

**Discussion by Gamefriend996** [oobabooga/text-generation-webui#5834] [Q&A] [answered]
Hy all.
A question, I'm trying my first LoRA training. It's already by 120 hours, 4571 hours remaining. My question now, if I use the Interrupt button, will everything be lost?

Thanks in advance.

**Comment by Zhincore** [comment, marked as answer]
It will save, don't worry. You should've also set periodic saving in the UI

**Reply by Gamefriend996** [reply]
Thanks for the tip with periodic saving.
